app/assets/stylesheets/hyrax/_collection_types.scss

```diff
@@ -47,3 +47,7 @@
     }
   }
 }
+
+.collection_type_share_applies_to {
```

Comment:
the convention for CSS is to use dashes (rather than underscores) in class names.